test/unit/app/browser/exportBookmarksTest.js

```diff
@@ -113,8 +113,8 @@ describe('Bookmarks export', function () {
     let result = exporter.createBookmarkHTML(personal, other)
     let expected = fs.readFileSync('./test/fixtures/bookmarkExport.html', 'utf8')
 
-    result = result.replace(/\s+/g,' ')
-    expected = expected.replace(/\s+/g,' ')
+    result = result.replace(/\s+/g, ' ')
+    expected = expected.replace(/\s+/g, ' ')
```

Comment:
Fixing a lint error (my bad on this one- I committed from Windows which doesn't run the `npm run lint` prehook)